server/sandbox_remove.go

```diff
@@ -99,6 +101,9 @@ func (s *Server) RemovePodSandbox(ctx context.Context, req *pb.RemovePodSandboxR
 	if err := s.CtrIDIndex().Delete(podInfraContainer.ID()); err != nil {
 		return nil, fmt.Errorf("failed to delete infra container %s in pod sandbox %s from index: %v", podInfraContainer.ID(), sb.ID(), err)
 	}
+	if err := os.Remove(filepath.Join(s.Config().RuntimeConfig.ContainerExitsDir, podInfraContainer.ID())); err != nil && !os.IsNotExist(err) {
+		return nil, errors.Wrapf(err, "failed to remove pod infra container exit file %s", podInfraContainer.ID())
```

Comment:
I almost think this should just just be a warning, but I don't have a all the context behind it.

Reply:
I would also be inclined to make tihs (and the corresponding one for non-infra containers) to be a warning

Reply:
The most usual case here is the file already being deleted and we cover that in the precheck.